040-SDK/130-transaction.mdx

```diff
@@ -334,24 +397,187 @@ result, _ := recordsClient.Transaction(context.TODO(), xata.TransactionRequest{
+      Id:    xata.String("rec_cfl4g6v00056"),
+    }),
+    // You can add additional transactions
+  }
+})
+```
+
+```jsonc
+// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/transaction
+{
+  "operations": [
+    {
+      "delete": {
+        "table": "titles",
+        "id": "rec_cfl4g6v00023"
+      }
+    },
+    {
+      "delete": {
+        "table": "titles",
+        "id": "rec_cfl4g6v00056"
+      }
+    }
+    // You can add additional transactions
+  ]
+}
+```
+
+</TabbedCode>
+
+### get
+
+Use `get` to fetch multiple records using IDs. It can retrieve records generated within the same transaction. If no record is found, the operation does not result in a canceled transaction.
+
+<TabbedCode tabs={['TypeScript', 'Python', 'Go', 'JSON']}>
+
+```ts
+const result = await xata.transactions.run([
+  {
+    get: {
+      table: 'titles',
+      id: 'rec_cfl4g6v00023',
+      columns: ['id', 'originalTitle']
+    }
+  },
+  {
+    get: {
+      table: 'titles',
+      id: 'rec_cfl4g6v00056',
+      columns: ['id', 'originalTitle']
+    }
+  }
+]);
+```
+
+```python
+result = xata.records().transaction({
+  "operations": [
+    {
+      "get": {
+        "table": "titles",
+        "id": "rec_cfl4g6v00023",
+        "columns": ["id", "originalTitle"]
+      }
+    },
+    {
+      "get": {
+        "table": "titles",
+        "id": "rec_cfl4g6v00056",
+        "columns": ["id", "originalTitle"]
+      }
+    },
+  ]
+})
+```
+
+```go
+recordsClient, _ := xata.NewRecordsClient()
+result, _ := recordsClient.Transaction(context.TODO(), xata.TransactionRequest{
+  Operations: []xata.TransactionOperation{
+    xata.NewGetTransaction(xata.TransactionGetOp{
+      Table:   "titles",
+      Id:      xata.String("rec_cfl4g6v00023"),
+      Columns: []string{"id", "originalTitle"},
+    }),
+    xata.NewGetTransaction(xata.TransactionGetOp{
+      Table:    "titles",
+      Id:       xata.String("rec_cfl4g6v00056"),
+      Columns: []string{"id", "originalTitle"},
+    }),
+  },
+})
+```
+
+```jsonc
+// POST https://{workspace}.{region}.xata.sh/db/{db}:{branch}/transaction
+{
+  "operations": [
+    {
+      "get": {
+        "table": "titles",
+        "id": "rec_cfl4g6v00023",
+        "columns": ["id", "originalTitle"]
+      }
+    },
+    {
+      "get": {
+        "table": "titles",
+        "id": "rec_cfl4g6v00056",
+        "columns": ["id", "originalTitle"]
+      }
+    }
+  ]
+}
+```
+
+</TabbedCode>
 
 ## Limitations
 
 - A maximum of 1000 operations are allowed in a single transaction. If you exceed this limit, you will receive an error message.
-- Inserting and updating content to file and file\[] (file array) columns is not permitted with transactions.
+- Inserting and updating content to `file` and `file\[]` (file array) columns is not permitted with transactions.
+- To prevent timeouts with the SQL proxy during transaction deletions, delete records in small batches. Each batch should not exceed 10,000 records.
```

Comment:
```suggestion
- To prevent timeouts with the SQL proxy during delete operations with a transaction, delete records in small batches. Each batch should not exceed 10,000 records.
```